components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/TiersApiService.java

```diff
@@ -17,5 +17,13 @@
 public abstract class TiersApiService {
     public abstract Response tiersGet(String accept,String ifNoneMatch)
     throws NotFoundException;
+    public abstract Response tiersPost(TierDTO body,String contentType)
```

Comment:
Please note that these files get auto-generated using the swagger-to-cxf maven plugin we use for REST API development.